packages/odyssey-react/src/components/Tag/Tag.module.scss

```diff
@@ -12,20 +12,20 @@
 
 .list {
   display: inline-block;
-  margin-block-end: -$spacing-xs;
```

Comment:
I looked at other components to validate but, is this the correct replacement? `-$spacing-xs` -> `SpaceRemXs`

Reply:
yes that is correct!